### PR TITLE
Adding id to the notifications payloads

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: server/android_notification_server.go [hunk 1]
```diff
@@ -43,6 +43,7 @@ func (me *AndroidNotificationServer) SendNotification(msg *PushNotification) Pus
 			"override_icon_url": msg.OverrideIconUrl,
 			"from_webhook":      msg.FromWebhook,
 			"version":           msg.Version,
+			"id":                msg.Id,
```
Comment (enahum) [inline server/android_notification_server.go]: can we rename this to `notificationId` ? see https://github.com/wix/react-native-notifications/blob/master/docs/advancedIos.md

Comment (jespino) [reply server/android_notification_server.go]: sure

File: server/server.go [hunk 2]
```diff
@@ -153,6 +156,43 @@ func handleSendNotification(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func handleAckNotification(w http.ResponseWriter, r *http.Request) {
+	ack := PushNotificationAckFromJson(r.Body)
+
+	if ack == nil {
+		rMsg := LogError("Failed to read ack body")
+		w.Write([]byte(rMsg.ToJson()))
+		incrementBadRequest()
```
Comment (enahum) [inline server/server.go]: FYI: I've gonna make changes to this

**Comment by enahum** [comment]
DO NOT MERGE THIS YET!

**Comment by hanzei** [comment]
@enahum What is blocking this PR from being merged?

**Comment by enahum** [comment]
Going to close this PR in favor of the work that I’m doing

**Comment by enahum** [comment]
See https://mattermost.atlassian.net/browse/MM-14858 if you want to follow up

**Pull request closed without merging**